src/features/PeopleDetails/index.js

```diff
@@ -52,7 +52,7 @@ export const PeopleDetails = () => {
         const month = date.split("-")[1];
         const year = date.split("-")[0];
         const formattedDate = day + "." + month + "." + year;
-        return formattedDate    
+        return formattedDate
```

Comment:
Here, I also suggest adjusting the function name to the camelCase convention and possibly writing it like this:
`const formatDate = (date) => {
    const [year, month, day] = date.split("-");
    return `${day}.${month}.${year}`;
}
`

or
`const formatDate = (date) => {
    if (!date) return "";
    const [year, month, day] = date.split("-");
    return `${day}.${month}.${year}`;
}
`